shift_detector/precalculations/TextMetadata.py

```diff
@@ -319,7 +292,7 @@ def metadata_function(self, language, words):
             if len(words) == 0:
                 return 0.0
             for word in words:
-                if word.lower() in stop:
+                if word in stop:
```

Comment:
I know you didn't touch this, but 'stop' is very undescriptive. Can you replace it with a more descriptive variable name?